grails-app/assets/javascripts/head.js

```diff
@@ -1 +1 @@
-//= require jquery-1.12.4.js
\ No newline at end of file
+//= require jquery-3.4.1.min.js
```

Comment:
is jquery 3 essential for this ? 

i propose we can avoid using jquery entirely for this login/logout and therefore avoid the likely versioning issues the apps will hit with this migration to jquery 3.